lang/cs/Org.Apache.REEF.IMRU/OnREEF/MapInputWithControlMessage/MapInputWithControlMessageCodec.cs

```diff
@@ -32,7 +32,7 @@ namespace Org.Apache.REEF.IMRU.OnREEF.MapInputWithControlMessage
     /// <typeparam name="TMapInput"></typeparam>
     internal sealed class MapInputWithControlMessageCodec<TMapInput> : IStreamingCodec<MapInputWithControlMessage<TMapInput>>
     {
-        private static Logger Logger = Logger.GetLogger(typeof(MapInputWithControlMessage<>));
+        private static readonly Logger Logger = Logger.GetLogger(typeof(MapInputWithControlMessage<>));
```

Comment:
This is a positive change, but unrelated to the change at hand. Hence, I'd suggest removing it.